serverlessworkflow/modules/ROOT/pages/persistence/postgresql-flyway-migration.adoc

```diff
@@ -0,0 +1,101 @@
+= Migrating Your PostgreSQL Database with a Kogito upgrade
+:flyway_install_url: https://flywaydb.org/download/community
+:flyway_migrate_existing_url: https://flywaydb.org/documentation/learnmore/existing
+:kogito_ddl_script_url: https://repository.jboss.org/org/kie/kogito/kogito-ddl
```

Comment:
we should provide links to Maven Central: https://repo1.maven.org/maven2/org/kie/kogito/kogito-ddl